onnxruntime/python/tools/transformers/optimizer.py

```diff
@@ -160,7 +161,7 @@ def optimize_by_fusion(
      Returns:
         object of an optimizer class.
     """
-    if model_type not in ["bert", "unet", "vae", "clip"] and (num_heads == 0 or hidden_size == 0):
+    if model_type not in ["bert", "swin", "unet", "vae", "clip"] and (num_heads == 0 or hidden_size == 0):
```

Comment:
add "vit"

Reply:
For vit, this num_heads and hidden_size are valid.

For swin, num_heads is a list, so it doesn't work, I set it to zero and add the exclusion here.

Reply:
I mean that vit could detect num_heads and hidden_size in fusion automatically?  If so, it is optional to specify them in command line.